tacs/mphys/builder.py

```diff
@@ -255,9 +278,35 @@ def get_coupling_group_subsystem(self, scenario_name=None):
         )
 
     def get_mesh_coordinate_subsystem(self, scenario_name=None):
+        """
+        The subsystem that contains the subsystem that will return the mesh
+        coordinates
+
+        Parameters
+        ----------
+        scenario_name : str or None
+            The name of the scenario calling the builder.
+
+        Returns
+        -------
+        mesh : :class:`~openmdao.api.Component` or :class:`~openmdao.api.Group` or None
+            The openmdao subsystem that has an output of coordinates.
+        """
         return TacsMeshGroup(fea_assembler=self.fea_assembler)
 
     def get_pre_coupling_subsystem(self, scenario_name=None):
+        """
+        Method that returns the openmdao subsystem to be added to each scenario before the coupling group
+
+        Parameters
+        ----------
+        scenario_name : str or None
```

Comment:
This is nitpicky, but I think the numpy standard for documenting optional parameter types is like [this](https://github.com/numpy/numpy/blob/d315bb0def0352da79d30327e7f65afd0b0046f6/numpy/core/einsumfunc.py#L1039-L1040)

So this would become:
```
scenario_name : str, optional
    The name of the scenario calling the builder, by default None, in which case ...
```

Reply:
Thanks, should be addressed now